packages/explorer/src/app/api/rows/route.ts

```diff
@@ -0,0 +1,33 @@
+import _ from "lodash";
+import { getDatabase } from "../database";
+
+export const dynamic = "force-dynamic";
+
+type Row = {
+  [key: string]: string;
+};
+
+type RowsResponse = Row[] | undefined;
+
+function convertKeysToCamelCase(rows: RowsResponse): Row[] {
+  if (rows == undefined) {
+    return [];
+  }
+
+  return rows.map((row) => {
+    return Object.keys(row).reduce((result: { [key: string]: string }, key) => {
+      const camelKey = _.camelCase(key);
```

Comment:
It's because when value schema is defined in camelCase e.g.

{"createdAt":"uint256","completedAt":"uint256","description":"string"}}

and when I fetch table schema, the keys are snake cased.

![CleanShot 2024-08-09 at 12 35 35@2x](https://github.com/user-attachments/assets/1b9d7d1e-b7a1-4b17-89f7-b07f0fd7a275)

Then, when updating the content in `EditableTableCell` I am using column's name to determine which field is being updated which couldn't work due to differing keys (one snake cased, one camel cased). I simplified the approach by only camel casing the column key when updating the value so I can determine the correct key (https://github.com/latticexyz/mud/pull/3001/files#diff-824dfa3431242e4dabe23c890ddf66a1c1058a22e9345b840a848938f3c72272R41) rather than camel casing all columns like before.